1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,42 @@
-function setAlarm() {}
+let isPaused = false;
+let timer;
+let mini;
+let seconds;
 
-// DO NOT EDIT BELOW HERE
```

Comment:
Good practice to place all your additions above this line. Not critical but helps to see what CYF have given